proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -89,6 +90,41 @@ message SearchResult {
   }
 }
 
+enum SearchContentFilter {
+  SEARCH_CONTENT_FILTER_UNSPECIFIED = 0;
+  SEARCH_CONTENT_FILTER_NAME = 1;
```

Comment:
What is name in this case? The name of the field, the name of the package, etc. ?

Reply:
oops I renamed these things and forgot to also rename the filters, updated and added some comments